Sources/SwiftDocCUtilities/Action/Actions/MergeAction.swift

```diff
@@ -40,18 +40,15 @@ struct MergeAction: Action {
         }
         var combinedJSONIndex = try JSONDecoder().decode(RenderIndex.self, from: jsonIndexData)
         
+        // Ensure that the destination has a data directory in case the first archive didn't have any pages.
```

Comment:
I don't think we can move this `createDirectory` call outside of the loop.

When I run a test merge command on two archives, I get an error:

```
swift run docc merge /path/to/SlothCreator.doccarchive /path/to/SymbolKit.doccarchive --output-path=/path/to/output.doccarchive

Error: The file “symbolkit” doesn’t exist.
```

The problem is the call to `copyItem` for this file: `/path/to/SymbolKit.doccarchive/documentation/symbolkit`... trying to copy to `/var/folders/some/temp/path/documentation/symbolkit`. The previous version of this code called `createDirectory` inside the loop, insuring that each target directory, such as `.../documentation/symbolkit`, exists before we try to copy files to it. By moving the call to `createDirectory` up outside the loop and only calling it once for the `data` directory, we aren't guaranteed that each target directory will exist.

Reply:
Can you tests that again or provide steps to reproduce this? I tried this on commit `66e4229` (the main merge from yesterday) with combinations of 1, 2, 3, and 4 archives with and without and explicit output path and I never encountered that error.

Reply:
The same two archives (SlothCreator and SymbolKit) in the same order also works for me.
```sh 
$ find output.doccarchive/data -maxdepth 2

output.doccarchive/data
output.doccarchive/data/documentation
output.doccarchive/data/documentation/symbolkit.json
output.doccarchive/data/documentation/symbolkit
output.doccarchive/data/documentation/slothcreator
output.doccarchive/data/documentation/slothcreator.json
output.doccarchive/data/tutorials
output.doccarchive/data/tutorials/slothcreator
output.doccarchive/data/tutorials/slothcreator.json
```